skbio/alignment/tests/test_tabular_msa.py

```diff
@@ -342,11 +345,34 @@ def test_keys_deleter_no_keys(self):
         self.assertFalse(msa.has_keys())
 
     def test_keys_deleter_with_keys(self):
-        msa = TabularMSA([RNA('UUU'), RNA('AAA')], key=str)
+        msa = TabularMSA([RNA('UUU'), RNA('AAA')], minter=str)
         self.assertTrue(msa.has_keys())
         del msa.keys
         self.assertFalse(msa.has_keys())
 
+    def test_get_cached_minter_str(self):
```

Comment:
Can you name these three methods `test_minter_*` or `test_minter_getter_*` to match the naming scheme of the other property tests?